substrate/frame/revive/src/storage/meter.rs

```diff
@@ -347,6 +343,10 @@ where
 			_ => ContractState::Alive,
 		}
 	}
+
+	pub(crate) fn take_all(&self) -> RawMeter<T, E, Nested> {
+		RawMeter { limit: self.available(), ..Default::default() }
+	}
```

Comment:
Same remarks as in the other comment in `gas.rs`.

```suggestion
	pub fn nestd_take_all(&self) -> RawMeter<T, E, Nested> {
		RawMeter { limit: self.available(), ..Default::default() }
	}
```